### PR TITLE
locality: blank=True, and errors raised for various scenarios:

### PR DESCRIPTION
**Pull request by goose-life**
- no locality given in spreadsheet (but working in one)
- locality given in spreadsheet (but working in a country)
- country doesn't match
- locality doesn't match

- errors not accidentally raised when all is in fact well

### CODE REVIEW

**Comment by goose-life** [comment]
see also https://github.com/longhotsummer/indigo-workflow/pull/20

File: indigo_app/views/works.py [hunk 1]
```diff
@@ -473,22 +473,23 @@ def get_table(self, spreadsheet_url):
     def get_frbr_uri(self, country, locality, row):
         frbr_uri = FrbrUri(country=row['country'], locality=row['locality'], doctype='act', subtype=row['subtype'], date=row['year'], number=row['number'], actor=None)
 
-        # check country matches, but ignore if no country given (model error raised)
+        # if the country doesn't match
+        # (but ignore if no country given – dealt with separately)
         if row['country'] and country.code != row['country'].lower():
-            raise ValueError('The country in the spreadsheet (%s: %s) doesn\'t match the country you\'re working in (%s: %s)' % (Country.objects.get(country_id=row['country']), row['country'], country, country.code.upper()))
+            raise ValueError('The country given in the spreadsheet (%s: "%s") doesn\'t match the country you\'re working in (%s: "%s")' % (Country.objects.get(country_id=row['country']), row['country'], country, country.code.upper()))
```
Comment (longhotsummer) [inline indigo_app/views/works.py]: Same issue with looking up Country: I can cause an error by providing a bad country id.

**Comment by longhotsummer** [comment]
LGTM.

**Comment by goose-life** [comment]
see issue #240

**Pull request merged**